build.sbt

```diff
@@ -95,16 +95,28 @@ lazy val jacksonParameterNamesJavacSettings = Seq(
 )
 
 // configuration of sbteclipse
+// Despite being from the Eclipse plugin these settings impact all the SBT project. As a consequence these default
+// values prevent you from using scala code in this sample app. Read the comments marked as 'SCALA SUPPORT'
+// to have sbt compile and use both java and scala source files.
+// DISCLAIMER: Enabling scala support has the side effect, in Eclipse IDE, to add a lot of clutter on the Project View.
```

Comment:
I think this could be more straightforward and to the point. Maybe something like this:

> Configures sbteclipse to enable Java support only, and disable Scala to remove clutter from the Project View.
> To use this project as a starting point for a mixed Java/Scala Lagom project, see the comments marked "SCALA SUPPORT" below.